cdc/sink/mysql/mysql.go

```diff
@@ -233,6 +229,7 @@ func (s *mysqlSink) EmitRowChangedEvents(ctx context.Context, rows ...*model.Row
 
 // FlushRowChangedEvents will flush all received events, we don't allow mysql
 // sink to receive events before resolving
```

Comment:
`we don't allow mysql
// sink to receive events before resolving` I don't know what does it mean. Can you make some improvement on this comment?

Reply:
Done